tests/test_routers/test_config.py

```diff
@@ -262,6 +262,11 @@ def test_update_config(app: FastAPI, wait_for_startup_after):
     get_config = client.get("/config").json()
     assert not get_config["pipelines"]
 
+    # Empty update
+    res = client.patch("/config", json={})
+    assert res.status_code == 200
+    assert get_config == client.get("/config").json()
```

Comment:
We already have `res` from the `patch`, with the current config.
```suggestion
    assert get_config == res.json()
```
Also, didn't we want to test that the config history didn't get longer?
```py
    with jsonlines.open(jsonl_file_path, "r") as reader:
        loaded_configs = list(reader)
    assert len(loaded_configs) == initial_config_count
```

Reply:
We have done the same throughout this test, to test that the config that we `get` after the `patch` is right (even though the patch returns it). 

I think that since we are testing once at the end of the test that the `config_history` length is right, we don't need to test it for just this case?

Reply:
> testing once at the end of the test

won't tell us which action increased the config history by one. This patch could start adding to the config history, and the following action could stop, at the same time. I agree it's unlikely, but tests are there for that.

Reply:
Ok, but at this point we were already testing 3 changes, and making sure at the end that only once did it save a new line - would you rather I change the logic so we test the length after each change? Or just for this one? 

Also, it will already be `initial_config_count + 1`, unless I move this piece of code to the beginning of the code. Would you rather I do that?

Reply:
Ok, let me know what you think of my last commit!